src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -826,6 +827,37 @@ result = file.getBytes()
 resultMetadata.put("content.type","image/png")
 ----
 
+=== Job Results
+
+Job Results are composed of two things: *Job Result List* and *Job Result Map*.
+Each task can contribute as for *Job Result List* as for *Job Result Map*.
+After job is finished, you can see Job Results in the Scheduler portal:
+
+image::scheduler-result-map.png[align=center]
+
+==== Job Result List
+
+Each task has its own result. However, some task results can be marked as *Task Result Added to Job Result*. In that case, this task result will be added to the *Job Result List*.
```

Comment:
`Each task has its own results`

Reply:
i believe task can have only single result. so I keep it singular

Reply:
Ok